setup.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup, find_packages, find_namespace_packages
```

Comment:
`find_packages` is unused here, I think.

Reply:
:+1: